### PR TITLE
connect api(s)

### PR DESCRIPTION
**Pull request by tech-bash**
Connected two api(s) for placement and mooc_courses....

### CODE REVIEW

File: src/components/TandPCell/CellComponent/Placements.js
Comment (anurag629) [inline src/components/TandPCell/CellComponent/Placements.js:1]: @tech-bash do not change the current CSS

**Review by anurag629 — CHANGES_REQUESTED** [review]
Use RTK Query from Redux Toolkit, just as we did for [AUTH](https://github.com/anurag629/IET_AGRA/blob/main/src/services/userAuthApi.js), to maintain the project structure.

File: src/components/TandPCell/CellComponent/Placements.js
Comment (tech-bash) [reply src/components/TandPCell/CellComponent/Placements.js:1]: Ok

**Comment by tech-bash** [comment]
> Use RTK Query from Redux Toolkit, just as we did for [AUTH](https://github.com/anurag629/IET_AGRA/blob/main/src/services/userAuthApi.js), to maintain the project structure.

ok

File: src/components/TandPCell/CellComponent/Placements.js [hunk 1]
```diff
@@ -1,68 +1,80 @@
...
-                            <td className="px-6 py-4">
-                                SDE
-                            </td>
-                            <td className="px-6 py-4">
-                                $1999
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
+                <Table>
```
Comment (anurag629) [inline src/components/TandPCell/CellComponent/Placements.js]: @tech-bash do not change the UI of the table, use same table

**Review by tech-bash — COMMENTED** [review]
Resolved

**Review by anurag629 — APPROVED** [review]
lgtm

**Pull request merged**